### PR TITLE
Registration for LARRY STUDIOS™

### PR DESCRIPTION
**Pull request by korn176**
**`UPDATED @ 2022-10-06 15:55 GMT+7` - Complied with contributers’ advice.**

Hello. I’m Korn, representative of LARRY STUDIOS™, a small game company located in Bangkok, Thailand. I’d like to register 3 (three) `is-a.dev` domains, which are `larryst.is-a.dev`, `larrystudios.is-a.dev`, and `nopply.is-a.dev`. All three are for LARRY STUDIOS™ and its affiliates. I have completed the application form below:

=========================================================

To make our job easier, please spend time to review your application before submitting. 

#### Add a x between [] if you meet that requirement example `[x]`
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. **This requirement can be raised if the domain you're registering is for emails**
  - Link to website: larrystudios.github.io, kornls.github.io, nopply.carrd.co, ranatown.github.io, *Arq091.github.io\* (under construction)*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain any slash.
- [x] There's enough information at the `owner` field.
   - You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.
 
 #### Feel free to join our discord server for more updates or any help related to the service :D - https://discord.gg/PZCGHz4RhQ

### CODE REVIEW

File: domains/fridge.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Fridge is-a.dev",
+  "owner": {
+    "username": "kornls",
+    "email": "larryst.korn@gmail.com"
+  },
+  "record": {
+    "CNAME": "Arq091.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/fridge.json:8]: We cannot add `fridge.is-a.dev` since the website hasm't been completed.

File: domains/kornls.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "KornLS is-a.dev",
+  "owner": {
+    "username": "kornls",
+    "email": "larryst.korn@gmail.com"
+  },
+  "record": {
+    "CNAME": "kornls.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/kornls.json:8]: Not enough content.

File: domains/rana.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Official Rana Town Website",
+  "owner": {
+    "username": "kornls",
+    "email": "larryst.korn@gmail.com"
+  },
+  "record": {
+    "CNAME": "ranatown.github.io"
```
Comment (WilliamDavidHarrison) [inline domains/rana.json:8]: Not enough content and the website needs to be in english.

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please refer to the review comments.

File: domains/fridge.json (hunk 1 above)
Comment (kornls) [reply domains/fridge.json:8]: Alright. I will inform you when it is completed. For now, I’ll remove the json file.

**Comment by kornls** [comment]
Thank you for your advice—I have complied accordingly, and deleted the three subdomains that you had requested.

**Review by wdhdev — APPROVED** [review]
Thank you for your quick response and quick resolution :)

Edit: I can't merge PR's incase you're wondering, as I'm only a helper.

**Review by andrewstech — APPROVED** [review]

**Pull request merged**